Bottom: 211afc2443ea9c1462f3e0b909def1f5a27af6be
Top:    9cf2ef88ac12caabad4e79c70563d1ead34e0d50
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-03 11:00:27 +0800

Refresh of tuntap-xdp-transmission

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index be6d993..7b8864e 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -3323,7 +3323,8 @@ struct socket *tun_get_socket(struct file *file)
 	struct tun_file *tfile;
 	if (file->f_op != &tun_fops)
 		return ERR_PTR(-EINVAL);
-	tfile = file->private_data;	if (!tfile)
+	tfile = file->private_data;
+	if (!tfile)
 		return ERR_PTR(-EBADFD);
 	return &tfile->socket;
 }
